src/ralph/ui/forms/addresses.py

```diff
@@ -388,6 +389,10 @@ def __init__(self, *args, **kwargs):
     widget = IPWithHostWidget()
 
     def compress(self, value):
+        if value:
+            hostname, ip_number = value
+            if not ip_number:
+                raise ValidationError(_('IP Address is required'))
```

Comment:
I prefer not to address tuples with index numbers as they are meaningless. I believe this is not overnested here.l